### PR TITLE
NLP page descriptions.

### PR DESCRIPTION
**Pull request by angela24680403**
This PR adds descriptions for NLP 1 and 2.
Screenshot 2024-01-08 at 14 52 31
Screenshot 2024-01-08 at 14 52 37

### CODE REVIEW

File: static/jc/the_search_for_anonymous_data.pdf
Comment (TheRootOf3) [inline static/jc/the_search_for_anonymous_data.pdf:1]: Is there a reason why we're removing this file or is it a misclick?

**Review by TheRootOf3 — CHANGES_REQUESTED** [review]
Please see the comment for the deleted file, otherwise contents look good! 🚀

File: static/jc/the_search_for_anonymous_data.pdf
Comment (angela24680403) [reply static/jc/the_search_for_anonymous_data.pdf:1]: I don't know what this file is, but I've moved it back.

**Review by TheRootOf3 — APPROVED** [review]
LGTM, thanks! 🎉

**Pull request merged**